src/main/java/org/usfirst/frc4904/robot/commands/controlpanel/SpinPanelThreeTimesColor.java

```diff
@@ -0,0 +1,39 @@
+package org.usfirst.frc4904.robot.commands.controlpanel;
+
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import org.usfirst.frc4904.robot.commands.controlpanel.StartPanelMotor;
+import org.usfirst.frc4904.robot.commands.controlpanel.StopPanelMotor;
+import java.util.function.Supplier;
+import org.usfirst.frc4904.standard.commands.RunUntil;
+import org.usfirst.frc4904.robot.RobotMap;
+
+public class SpinPanelThreeTimesColor extends ColorSensor {
+
+    private final Motor motor;
+
+    /**
+     * This command will spin the control panel three times based off the camera
+     * feed.
+     * 
+     * @param motor The motor to spin the control panel.
+     */
+
+    public SpinPanelThreeTimesColor(String name) {
+        super();
+        this.motor = RobotMap.Component.controlPanel;
+        addRequirements(this.motor);
+        setName(name);
+
+        Supplier<Boolean> isDone = () -> {
+            trackColorChange();
```

Comment:
is there a reason why `trackColorChange` can't return `doneSpinning` instead of just setting it in `ColorSensor`